### DISCUSSION TITLE
Installing missing packages in the current buffer

### DISCUSSION THREAD

**Discussion by PMassicotte** [R-nvim/R.nvim#158] [Ideas]
Made this new module to automatically install reported missing packages in the current buffer.

![Peek 2024-06-26 12-15](https://github.com/R-nvim/R.nvim/assets/4519221/4bae9fe0-777c-4848-a473-891e19f5ba37)

**Comment by jalvesaq** [comment]
Great! This is something that I will use. I never install R. I build it from source when a new version is released and update the symbolic links to `R` and `Rscript` at `~/bin` and, then, I have to install all packages again.

**Comment by PMassicotte** [comment]
yeah same! 

Also happen when I open student's scripts and I am missing some packages that I have to install manually.
